tasks/change-granule-collection-s3/schemas/config.json

```diff
@@ -107,6 +107,10 @@
     "invalidGranuleBehavior": {
       "type": ["string", "null"],
       "description": "what to do if a granule without valid details (file buckets and keys) is ingested."
+    },
+    "cmrGranuleUrlType": {
+      "type": ["string", "null"],
+      "description": "The type of URL to use for granule links in CMR metadata `(['http', 's3', 'both'])`.   Defaults to `both`."
```

Comment:
How did you arrive at 'both' as the default?

Reply:
It's the default in `update-granules-cmr-metadata-file-links`  and a couple of other places in the code, most importantly `updateCMRMetadata`, which is what this is passed to.